igor_src/main.cpp

```diff
@@ -252,6 +268,14 @@ int main(int argc , char* argv[]){
 			// End the program without error
 			return 0;
 		}
+		
+		//Command line argument asking for igor datadir
+		if(string(argv[carg_i]) == string("-datadir")){
+			// Display IGoR's version
+			cout<<IGOR_DATA_DIR<<endl;
+			// End the program without error
```

Comment:
```suggestion
		if(string(argv[carg_i]) == string("-getdatadir")){
			// Print IGoR's data directory path
			cout<<IGOR_DATA_DIR<<endl;
			// End the program without error
```
I think it would be better to make it clear that this command cannot be used to set a datadir using the command's name
An alternative would be to do the following:
- igor -datadir [+- -otherargs] prints the datadir
- igor -datadir mypathblabla [+- -otherargs] sets the datadir to mypathblabla